sdk/resourcemanager/Azure.ResourceManager/src/Extensions/ArmClientBuilderExtensions.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core.Extensions;
+using Azure.ResourceManager;
+
+namespace Microsoft.Extensions.Azure
+{
+    /// <summary>
+    /// Extension methods to add <see cref="ArmClientOptions"/> client to clients builder.
```

Comment:
I noticed that I refer to the wrong type here. When I fix it locally, and run `.\eng\scripts\CodeChecks.ps1 -ServiceDirectory resourcemanager`, I get a whole bunch of warnings. Is it safe for me to push a change that corrects this error, or will I end up messing up my PR then?
```
Validating installation instructions
git diff
diff --git a/sdk/resourcemanager/Azure.ResourceManager/src/Extensions/ArmClientBuilderExtensions.cs b/sdk/resourcemanager/Azure.ResourceManager/src/Extensions/ArmClientBuilderExtensions.cs
index 89530144e9..68d87da0d8 100644
--- a/sdk/resourcemanager/Azure.ResourceManager/src/Extensions/ArmClientBuilderExtensions.cs
+++ b/sdk/resourcemanager/Azure.ResourceManager/src/Extensions/ArmClientBuilderExtensions.cs
@@ -7,7 +7,7 @@
 namespace Microsoft.Extensions.Azure
 {
     /// <summary>
-    /// Extension methods to add <see cref="ArmClientOptions"/> client to clients builder.
+    /// Extension methods to add <see cref="ArmClient"/> client to clients builder.
     /// </summary>
     public static class ArmClientBuilderExtensions
     {
error: Generated code is not up to date.
    You may need to rebase on the latest main,
    run 'eng\scripts\Update-Snippets.ps1' if you modified sample snippets or other *.md files (https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#updating-sample-snippets),
    run 'eng\scripts\Export-API.ps1' if you changed public APIs (https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#public-api-additions).
    run 'dotnet build /t:GenerateCode' to update the generated code.
    run 'dotnet build /t:GenerateTests' to update the generated test code.

To reproduce this error locally, run 'eng\scripts\CodeChecks.ps1 -ServiceDirectory resourcemanager'.

Summary:

   1 error(s)

error : Generated code is not up to date.
    You may need to rebase on the latest main,
    run 'eng\scripts\Update-Snippets.ps1' if you modified sample snippets or other *.md files (https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#updating-sample-snippets),
    run 'eng\scripts\Export-API.ps1' if you changed public APIs (https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#public-api-additions).
    run 'dotnet build /t:GenerateCode' to update the generated code.
    run 'dotnet build /t:GenerateTests' to update the generated test code.

To reproduce this error locally, run 'eng\scripts\CodeChecks.ps1 -ServiceDirectory resourcemanager'.
```

Reply:
You will need to run `eng\scripts\Export-Api.ps1 resourcemanager`

Reply:
I did that (before, and now again), and it completed successfully, without generating additional local changes:
```
PS C:\Source\per-azure-sdk-for-net> .\eng\scripts\Export-API.ps1 resourcemanager
MSBuild version 17.5.1+f6fdcf537 for .NET
  Determining projects to restore...
  All projects are up-to-date for restore.
  Azure.ResourceManager -> C:\Source\per-azure-sdk-for-net\artifacts\bin\Azure.ResourceManager\Release\netstandard2.0\Azure.ResourceManager.dll
  Generated reference assembly source code: C:\Source\per-azure-sdk-for-net\sdk\resourcemanager\Azure.ResourceManager/api/Azure.ResourceManager.netstandard2.0.cs

Build succeeded.
    0 Warning(s)
    0 Error(s)

Time Elapsed 00:00:02.67
```
Still, `eng\scripts\CodeChecks.ps1 -ServiceDirectory resourcemanager` fails with the same error as before.

I created a draft PR with this fix to the main repo, and it got through all checks. This Should™️be a sound fix.